### PR TITLE
enable wistia embeds

### LINKED ISSUE #65: add support for Wistia video embeds
enable embeds of videos hosted on Wistia using video codes!

### PR DESCRIPTION
**Pull request by MarquiseRosier**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #65 

Test URLs:
- Before: https://main--shredit--stericycle.aem.page/en-us/resource-center/videos/decluttering-your-workspace
- After: 
  - https://wistia-embeds--shredit--stericycle.aem.page/en-us/resource-center/videos/decluttering-your-workspace
  - https://wistia-embeds--stericycle-shared--aemsites.aem.page/en-us/resource-center/videos/decluttering-your-workspace

### CODE REVIEW

**Review by kronnox — APPROVED** [review]

File: blocks/video/video.js [hunk 1]
```diff
@@ -52,6 +52,23 @@ function embedVimeo(url, replacePlaceholder, autoplay) {
...
+  let suffix = '';
+  if (replacePlaceholder || autoplay) {
+    const suffixParams = {
+      autoplay: '1',
+      background: autoplay ? '1' : '0',
+    };
+    suffix = `?${Object.entries(suffixParams).map(([k, v]) => `${k}=${encodeURIComponent(v)}`).join('&')}`;
+  }
+  const temp = document.createElement('div');
+  temp.innerHTML = `<div style="left: 0; width: 100%; height: 0; position: relative; padding-bottom: 56.25%;">
```
Comment (seanathero) [inline blocks/video/video.js:65]: Any reason we can't use a css class here instead of inline styles?

Comment (dkuntze) [reply blocks/video/video.js:65]: I agree with @seanathero here @MarquiseRosier. Easy change.

Comment (dkuntze) [reply blocks/video/video.js:65]: Plus it'll give you a chance to size the video player fill up the width of the content area.

like this:
image

**Comment by MarquiseRosier** [comment]
@dkuntze @seanathero thanks for the feedback; does this work :)

File: blocks/video/video.js [hunk 2]
```diff
@@ -25,7 +25,7 @@ function embedYoutube(url, replacePlaceholder, autoplay) {
   }
 
   const temp = document.createElement('div');
-  temp.innerHTML = `<div style="left: 0; width: 100%; height: 0; position: relative; padding-bottom: 56.25%;">
+  temp.innerHTML = `<div>
       <iframe src="https://www.youtube.com${vid ? `/embed/${vid}?rel=0&v=${vid}${suffix}` : embed}" style="border: 0; top: 0; left: 0; width: 100%; height: 100%; position: absolute;"
```
Comment (seanathero) [inline blocks/video/video.js:29]: should we also use a class here instead of inline style, since we're making that change later in the file?

**Comment by MarquiseRosier** [comment]
@dkuntze sorry about that; please check again when you can!

**Pull request merged**